src/components/index.tsx

```diff
@@ -39,7 +39,8 @@ import { ProgressStep } from './ProgressStep'
 import { AvatarGroup, Avatar } from './Avatar'
 import { Pagination } from './Pagination'
 import { Table } from './Table'
-
+import NavigationItem from './NavigationItem'
```

Comment:
"If NavigationItem is not a shared component, I suggest importing it from the SideNavigationFolder along with SideNavigation.

Reply:
@KaroGhulyan resolved!